apps/balancer-tools/src/app/poolsimulator/(components)/PoolParamsForm.tsx

```diff
@@ -8,21 +8,19 @@ import { Input } from "#/components/Input";
 import { Form, FormField } from "#/components/ui/form";
 import {
   AnalysisData,
-  PoolTypeEnum,
   usePoolSimulator,
 } from "#/contexts/PoolSimulatorContext";
-import {
-  ECLPSimulatorDataSchema,
-  StableSwapSimulatorDataSchema,
-} from "#/lib/schema";
+import { ECLPSimulatorDataSchema, PoolSimulatorDataSchema } from "#/lib/schema";
+
+import { CombinedParams, PoolTypeEnum } from "../(types)";
 
 const schemaMapper = {
-  MetaStable: StableSwapSimulatorDataSchema,
-  GyroE: ECLPSimulatorDataSchema,
+  [PoolTypeEnum.MetaStable]: PoolSimulatorDataSchema,
```

Comment:
I changed this on my PR, think that the name is not that clear

Reply:
which PR?

Reply:
oh ok got it now `PoolSimulatorDataSchema` now is `StableSwapSimulatorDataSchema`. Thanks!